indico/modules/events/contributions/testing/fixtures.py

```diff
@@ -0,0 +1,39 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from datetime import timedelta
+
+import pytest
+
+from indico.modules.events.contributions.models.contributions import Contribution
+
+
+@pytest.fixture
+def create_contribution(db, dummy_event):
+    """Returns a a callable that lets you create a contribution"""
+
+    def _create_entry(title, duration):
+        entry = Contribution(event_new=dummy_event.as_event, title=title, duration=duration)
+        db.session.add(entry)
+        db.session.flush()
+        return entry
+
+    return _create_entry
+
+
+@pytest.fixture
+def dummy_contribution(create_contribution):
+    return create_contribution(title="Dummy Contribution", duration=timedelta(minutes=20))
```

Comment:
this is a string that goes into the db so it makes sense to make it unicode.

the reason for not using `unicode_literals` in the actual test files is mostly parametrized tests (the "test ids" are pretty awful when using unicode strings instead of bytestrings.. dunno if it is fixed by now or not)